### PR TITLE
Added new certs to v5

### PR DESCRIPTION
**Pull request by stechiu**
### Summary of changes

 - This PR contains the new certs 

### Checklist

 - [ ] Added a changelog entry
 - [ ] Relevant test coverage

### Authors
@stechiu

### CODE REVIEW

File: BraintreeCore/src/main/java/com/braintreepayments/api/core/TLSCertificatePinning.kt [hunk 1]
```diff
@@ -1,91 +1,718 @@
-package com.braintreepayments.api.core
+package com.braintreepayments.api
```
Comment (saperi22) [inline BraintreeCore/src/main/java/com/braintreepayments/api/core/TLSCertificatePinning.kt:1]: package name has changed in v5 - please put back the `.core` suffix.

Comment (stechiu) [reply BraintreeCore/src/main/java/com/braintreepayments/api/core/TLSCertificatePinning.kt]: Done

File: BraintreeCore/src/main/java/com/braintreepayments/api/core/TLSCertificatePinning.kt [hunk 2]
```diff
@@ -5,87 +5,714 @@ import java.io.InputStream
...
+ * 'braintree-mobile-certificate-generation'. This script concatenates '.crt' files in the 'certificates'
+ * directory.
+ *
+ * Encapsulates our trusted x509 Certificates for Secure SSL Communication with Braintree's servers
+ * to avoid storing certificate files in an 'NSBundle'.
+ *
+ * Do not update this file directly. If new certificates are needed, update the 'certificates' directory
+ * in 'braintree-mobile-certificate-generation' and re-run the certificate generation script.
+ */
+@Suppress("LargeClass")
```
Comment (tdchow) [inline BraintreeCore/src/main/java/com/braintreepayments/api/core/TLSCertificatePinning.kt:19]: Did you have to manually add this suppress annotation? Maybe we can tweak the detekt rule to exclude this class, so the next time we generate the cert, we don't need to have any manual steps.

Comment (stechiu) [reply BraintreeCore/src/main/java/com/braintreepayments/api/core/TLSCertificatePinning.kt:19]: `@Suppress` was added to the new bash script for the `LargeClass` linter error. You can see the script in this PR https://github.braintreeps.com/braintree/braintree-mobile-certificate-generation/pull/7

The script hasn't been merged yet. If there are changes we should make, we can do so in the script PR and I'll generate an updated file to add to Android BT SDK

Comment (stechiu) [reply BraintreeCore/src/main/java/com/braintreepayments/api/core/TLSCertificatePinning.kt:19]: Also how do we exclude the class?

Comment (tdchow) [reply BraintreeCore/src/main/java/com/braintreepayments/api/core/TLSCertificatePinning.kt:19]: Oh nice! I didn't realize the script was adding the suppress annotation.

You can modify the `detekt-config.yml` file to change the ruleset: https://detekt.dev/docs/introduction/configurations#path-filters--excludes--includes

Comment (stechiu) [reply BraintreeCore/src/main/java/com/braintreepayments/api/core/TLSCertificatePinning.kt:19]: Thanks :) I made the changes in `detekt-config.yml`

**Review by tdchow — APPROVED** [review]

**Review by saperi22 — APPROVED** [review]

File: detekt/detekt-config.yml [hunk 3]
```diff
@@ -107,6 +107,7 @@ complexity:
   LargeClass:
     active: true
     threshold: 600
+    excludes: 'TLSCertificatePinning.kt'
```
Comment (saperi22) [inline detekt/detekt-config.yml:110]:
```suggestion
    excludes: ['**/TLSCertificatePinning.kt']
```

Comment (saperi22) [reply detekt/detekt-config.yml:110]: `TLSCertificatePinning.kt` is not in the same directory as the detekt config. We'd need the absolute path or use a ** to look for all directories for the file.

Comment (stechiu) [reply detekt/detekt-config.yml]: Ohh I see, I couldn't figure out why tests were failing yesterday. Appreciate the help!

**Pull request merged**